.github/workflows/build_osgeo4w.sh

```diff
@@ -111,7 +112,7 @@ cp -a $(ldd $dist/lib/*.dll | awk '/mingw64/{print $3}' |
         mswindows/osgeo4w/env.bat.tmpl
     cat <<EOT
 
-set PATH=%OSGEO4W_ROOT%\\apps\\msys\\bin;%PATH%
+rem set PATH=C:\\msys64\\mingw64\\bin;C:\\msys64\\usr\\bin;%OSGEO4W_ROOT%\\apps\\msys\\bin;%PATH%
```

Comment:
```suggestion
set PATH=%OSGEO4W_ROOT%\\bin;%PATH%

IF EXIST "C:\msys64/usr/bin/" set PATH=%PATH%;C:/msys64/usr/bin/
IF EXIST "C:\msys64/usr/bin/" set PATH=%PATH%;C:/msys64/usr/bin/

echo %PATH%

```
set and print path for debugging...